CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 2016-06-01 Unsupported Release 0.7.0
+
+ * Manage sources - `chocolateysource` - see [MODULES-3037](https://tickets.puppetlabs.com/browse/MODULES-3037)
+ * $::chocolateyversion fact is optional - see [#110](https://github.com/chocolatey/puppet-chocolatey/issues/110)
+ * Fix: puppet apply works again - see [#105](https://github.com/chocolatey/puppet-chocolatey/issues/105)
+
+# Original Module
```

Comment:
Will the term `Original` be confusing?  Perhaps `Original module published by Chocolatey` or something more descriptive?